### PR TITLE
Update Profile editor

### PR DESCRIPTION
**Pull request by bob2402**

### CODE REVIEW

File: app/UI/components/tw.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ export const inputBorderClass = `border-[2px] border-[${DividerBackgroundColor}]
 export const DividerClass = `h-[0.0625rem] bg-[${DividerBackgroundColor}] my-[1.5rem] w-full`;
 export const LinearGradientsClass = "bg-gradient-to-r from-[#FF762C] via-[#FF3A5E] to-[#FF01A9]";
 export const InputClass =
-    `w-full px-4 py-3 rounded-lg resize-y bg-transparent ${NoOutlineClass} ${inputBorderClass} placeholder-[${PlaceholderColor}] text-[${PrimaryTextColor}]`;
+    `w-full px-4 py-3 rounded-lg resize-y bg-transparent ${NoOutlineClass} ${inputBorderClass} placeholder-[${PlaceholderColor}] text-[${PrimaryTextColor}] focus:ring-0 focus:ring-[${PrimaryTextColor}] focus:border-[${PrimaryTextColor}] focus:border-[2px] focus:border-solid focus:border-opacity-100`;
```
Comment (BlowaterNostr) [inline app/UI/components/tw.ts:9]: 1. `InputClass` is used by other component as well. Do not change global dependencies just for a single page unless we have agreed on it at the design level.

Make a copy with the value you want. A little duplicate is better than a little dependency.

2. reused Global tw class set like this is a very very bad practice. We should not do it.
It's only good to define single reused class for semantics.

Styles should be defined local to the component.

File: app/UI/edit-profile.tsx [hunk 2]
```diff
@@ -49,134 +46,154 @@ export class EditProfile extends Component<Props, State> {
         profileData: undefined,
     };
 
+    newFieldKey = createRef<HTMLInputElement>();
+    newFieldValue = createRef<HTMLInputElement>();
```
Comment (BlowaterNostr) [inline app/UI/edit-profile.tsx:50]: maybe we don't need ref here.

**Review by BlowaterNostr — CHANGES_REQUESTED** [review]

File: app/UI/edit-profile.tsx (hunk 2 above)
Comment (bob2402) [reply app/UI/edit-profile.tsx:43]: They are used for the addField function.

Comment (BlowaterNostr) [reply app/UI/edit-profile.tsx:43]: What I meant was the addField function could be reached by states

**Pull request merged**